prep-javascript-loops/main.js

```diff
@@ -0,0 +1,74 @@
+
+function whileLoop1 () {
+  let newArray = []
```

Comment:
Arrays and objects should be declared with the `const` keyword since you can still update the values arrays and objects contain when they are declared with the const keyword. 

The use of let for `i` (loop index variables) is correct as the value of `i` will change during run time.